src/components/AddTask/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from "react";
+import "./index.css";
+import PriorityType from "../../types/PriorityType";
+import TaskType from "../../types/TaskType";
+
+interface AddTaskProps {
+  addTask: (task: TaskType) => void;
+}
+
+const AddTask = ({ addTask }: AddTaskProps) => {
+  const [title, setTitle] = useState("");
+  const [description, setDescription] = useState("");
+  const [priority, setPriority] = useState(PriorityType.low);
+
+  const handleAddTask = () => {
+    if (!title || !description) {
+      return;
+    }
+    const task: TaskType = {
+      id: `${Date.now()}-${Math.floor(Math.random() * 10000)}`,
+      title: title,
+      description: description,
+      priority: priority,
+      completed: false,
+    };
+    addTask(task);
+    setTitle("");
+    setDescription("");
+    setPriority(PriorityType.low);
+  };
+
+  const priorityColor = () => {
+    switch (priority) {
+      case PriorityType.high:
+        return "bg-color-red";
+      case PriorityType.medium:
+        return "bg-color-yellow";
+      case PriorityType.low:
+        return "bg-color-green";
+      default:
+        return "";
+    }
+  };
+
+  return (
+    <div className="add-task-panel">
+      <button
+        className="add-task-btn"
+        id="add-task-btn"
+        onClick={handleAddTask}
+      >
+        Add
+      </button>
+      <div className="add-task-info-panel">
+        <div className="add-task-info-header-panel">
+          <input
+            type="text"
+            className={"task-title" + (!title ? " input-error" : "")}
```

Comment:
I prefer to enable this error only once try to add task that as not title or description and not enable it always even though before submit the task